profile/README.md

```diff
@@ -8,8 +8,8 @@ Flow is a PHP-based, strongly typed ETL (Extract, Transform, Load) and asynchron
 [![Test Suite](https://github.com/flow-php/flow/actions/workflows/test-suite.yml/badge.svg?branch=1.x)](https://github.com/flow-php/flow/actions/workflows/test-suite.yml)
 
 - 📈 [Project Roadmap](https://github.com/orgs/flow-php/projects/1)
-- 📜 [Documentation](https://flow-php.comhttps://flow-php.com/documentation/)
-- 🛠️ [Contributing](https://github.com/flow-php/flow/blob/1.xhttps://flow-php.com/documentation/contributing.md)
+- 📜 [Documentation](https://flow-php.com/documentation/)
+- 🛠️ [Contributing](https://flow-php.com/documentation/contributing.md)
 - 🚧 [Upgrading](https://github.com/flow-php/flow/blob/1.x/UPGRADE.md)
```

Comment:
this link doesn't work...

Reply:
ah yeah, this was moved to /documentation folder in the monorepo 🤦‍♂️